languages/javascript/exercises/concept/basics/.meta/config.json

```diff
@@ -0,0 +1,15 @@
+{
+  "contributors": [
+    {
+      "github_username": "SleeplessByte",
+      "exercism_username": "SleeplessByte"
+    }
+  ],
```

Comment:
I've checked, and this can indeed be removed (no duplicate users). I'll be submitting a PR to update this soon.

```suggestion
 ```